packages/vscode-apollo/src/debug.ts

```diff
@@ -0,0 +1,74 @@
+import { window, workspace, OutputChannel } from "vscode";
+
+/**
+ * for errors (and other logs in debug mode) we want to print
+ * a stack trace showing where they were thrown. This uses an
+ * Error's stack trace, removes the three frames regarding
+ * this file (since they're useless) and returns the rest of the trace.
+ */
+const createAndTrimStackTrace = () => {
+  let stack: string | undefined = new Error().stack;
```

Comment:
Can `stack` be undefined if you're initializing it inline like this?

Reply:
Unfortunately. The type of `new Error().stack` can be undefined 😞